benchmarks/thermal_convection_3d/thermal_convection_3d.prm

```diff
@@ -0,0 +1,172 @@
+  end
+end
+
+# In our case, the gravity magnitude specifies the Rayleigh number.
+subsection Gravity model
+  set Model name = radial constant
+  subsection Radial constant
+    set Magnitude = 7e3
+  end
+end
+
+
+############### Parameters describing the temperature field
+# The governing equations describe an incompressible Boussinesq fluid
+# at infinite Prandtl number 3D spherical shell that is heated from
+# below and cooled from above.
+subsection Formulation
+  set Formulation = Boussinesq approximation
+end
+
+# The temperature scale is chosen as the difference between the
+# temperature at the inner and outer boundaries (equal to 1).
+subsection Boundary temperature model
+  set Fixed temperature boundary indicators   = bottom, top
+  set List of model names = spherical constant
+  subsection Spherical constant
+    set Inner temperature = 1
+    set Outer temperature = 0
+  end
+end
+
+# The initial temperature is given by the sum of a conductive profile
+# together with a perturbation. The perturbation determines the final
+# patterns of the polyhedral symmetry which is here cubic.
+subsection Initial temperature model
+  set Model name = harmonic perturbation CitcomS
+  subsection Harmonic perturbation CitcomS
+    # The vertical (radial or depth) wave number allows the user to prescribe a single up-
+    # / downswing or half periods. 
+    set Vertical wave number   = 1
+    # For the cubic initial condition temperature, the axisymmetric
+    # spherical harmonic is of degree l = 4 and order m = 0. 
+    set Lateral wave number l1 = 4
+    set Lateral wave number m1 = 0
+    # For the cubic initial condition temperature, the nonaxisymmetric
+    # spherical harmonic is of degree l = 4 and order m = 4. 
+    set Lateral wave number l2 = 4
+    set Lateral wave number m2 = 4
+    # The magnitude specifies the magitude of the perturbation.
+    set Magnitude              = 0.01
+    # A perturbation parameter has been introduced to slowly perturb
+    # the amplitude of the nonaxisymmetric mode. Set to 0 means the
+    # perturbation is full; set to 1 and there is no perturbation (i.e
+    # the initial condition temperature profile is only axissymmetric)
+    set Delta                  = 0
+  end
+end
+
+
+############### Parameters describing the discretization
+# The following parameters describe how often we want to refine the
+# mesh globally and adaptively, what fraction of cells should be
+# refined in each adaptive refinement step, and what refinement
+# indicator to use when refining the mesh adaptively.
+subsection Mesh refinement
+  set Initial global refinement   = 2
+  set Initial adaptive refinement = 0
+  #set Strategy                    = temperature
+end
+
+# In order to get similar results to CitcomS, two factors in the
+# entropy viscosity (cR) and artificial viscosity (beta) stabilization
+# have to be set to 0. See the 2012 paper by Kronbichler, Heister and
+# Bangerth that describes ASPECT for more information.
```

Comment:
Here the odd thing: removing either beta OR cR stabilization leads to same results in the average temperature than removing both stabilization parameters. However, using both stabilization parameters leads to a significantly different result (I performed this test a couple of time). I do not understand what is happening here but it can surely find an explanation in the mesh resolution I used: 
These test were performed using a level 4 global mesh refinement, which is quite low. A new test using a higher mesh resolution and with both stabilization is running now which should give similar results I hope. But it still seems odd to me that removing either beta or cR lead to exact same result, but not with removing both parameters..

![thermal_convection_stabilization-test](https://user-images.githubusercontent.com/32137895/44089708-1b1cff94-9fc8-11e8-91e7-2359c76c2394.png)

Reply:
![3dtcb_stabilization_average-temperature](https://user-images.githubusercontent.com/32137895/44409351-82860780-a562-11e8-9dd9-62dec16575e2.png)
Here a better figure with testing different level of mesh refinement. Higher mesh refinement leads to the accurate answer. Also the gap between these for level 4 refinement is quite huge.